backend/src/main/java/com/bakdata/conquery/commands/ManagerNode.java

```diff
@@ -165,18 +165,20 @@ public void loadNamespaces() {
 
 		loaders.shutdown();
 		while (!loaders.awaitTermination(1, TimeUnit.MINUTES)) {
-			final int coundLoaded = registry.getDatasets().size();
-			log.debug("Waiting for Worker namespaces to load. {} are already finished. {} pending.", coundLoaded, namespaceStorages.size()
-																												  - coundLoaded);
+			final int countLoaded = registry.getNamespaces().size();
+			log.debug("Waiting for Worker namespaces to load. {} are already finished. {} pending.", countLoaded, namespaceStorages.size()
+																												  - countLoaded);
 		}
 	}
 
 	private void loadMetaStorage() {
 		log.info("Opening MetaStorage");
-		getMetaStorage().openStores(getInternalMapperFactory().createManagerPersistenceMapper(getDatasetRegistry(), getMetaStorage()), getEnvironment().metrics());
-		log.info("Loading MetaStorage");
-		getMetaStorage().loadData();
-		log.info("MetaStorage loaded {}", getMetaStorage());
+		getMetaStorage().openStores(getInternalMapperFactory().createManagerPersistenceMapper(getDatasetRegistry(), getMetaStorage()));
+		if (getConfig().getStorage().isLoadStoresOnStart()) {
+			log.info("Loading MetaStorage");
+			getMetaStorage().loadData();
+			log.trace("MetaStorage loaded {}", getMetaStorage());
```

Comment:
```suggestion
			log.trace("DONE loading MetaStorage {}", getMetaStorage());
```

ggf MetaStorage doppelt?

Reply:
bevorzuge die DONE messages in DEBUG